### PR TITLE
Fix arguments for standard analysis with pp sample

### PR DESCRIPTION
**Pull request by fgrosa**

### CODE REVIEW

File: GetRawYieldsDplusDs.py [hunk 1]
```diff
@@ -334,6 +334,12 @@
                         hSigmaToFix.GetBinContent(iPt+1)+hSigmaToFix.GetBinError(iPt+1))
                 else:
                     print('WARNING: impossible to fix sigma! Wrong mult factor set in config file!')
+        else:
+            if hSigmaToFix:
+                massFitter[iPt].SetInitialGaussianMean(
+                    hSigmaToFix.GetBinContent(iPt+1)*fitConfig[cent]['SigmaMultFactor'])
+            else:
+                massFitter[iPt].SetInitialGaussianMean(0.008)
```
Comment (fcatalan92) [inline GetRawYieldsDplusDs.py:342]: Probably here you meant `SetInitialGaussianSigma`

File: systematics/seleff/PlotCutVariationsOnePtBin.cc [hunk 2]
```diff
@@ -100,13 +100,13 @@ void PlotCutVariationsOnePtBin(TString cfgFileName) {
                 hPromptFrac[iFile] = (TH1D*)hCrossSection[iFile]->Clone();
                 hFDFrac[iFile] = (TH1D*)hCrossSection[iFile]->Clone();
                 TGraphAsymmErrors* gFcConservative = (TGraphAsymmErrors*)infile_crossec->Get("gFcConservative");
-                for(int iPt=0; iPt<gFcConservative->GetN(); iPt++) {
+                for(int iPt=1; iPt<gFcConservative->GetN(); iPt++) {
                     double pT, promptFrac;
                     gFcConservative->GetPoint(iPt, pT, promptFrac);
```
Comment (fcatalan92) [inline systematics/seleff/PlotCutVariationsOnePtBin.cc:105]: Now `GetPoint` start from 1 and not from 0, is it intended?

File: RunFullAnalysis.sh [hunk 3]
```diff
@@ -228,10 +228,23 @@ if $DoHFPtSpec; then
     Channel="kDplusKpipi"
   fi
   
+  cc=""
+  year=""
+  sigma=1.
+  if [ ${Cent} == "k010" -o "${Cent}" == "k3050" -o "${Cent}" == "k3050" ]; then
```
Comment (fcatalan92) [inline RunFullAnalysis.sh:234]: Here `"k3050"` is present two times

File: systematics/seleff/PlotCutVariationsOnePtBin.cc (hunk 2 above)
Comment (fgrosa) [reply systematics/seleff/PlotCutVariationsOnePtBin.cc:105]: Yes it is correct, this is due to the fact that for some reason the point `0` is set to zero in `gFcConservative`, while it is filled starting from `1`.

File: RunFullAnalysis.sh (hunk 3 above)
Comment (fgrosa) [reply RunFullAnalysis.sh]: Thanks! Indeed this was a typo, fixed in f4df70a

File: GetRawYieldsDplusDs.py (hunk 1 above)
Comment (fgrosa) [reply GetRawYieldsDplusDs.py]: Thanks! Yes indeed this was a typo, fixed in 78c235e

**Pull request merged**